Create Step1

kses1010 opened the pull request:
Step1 기능/프로그래밍 요구사항에 적용하여 구현

들여쓰기 2단계 이내로 구현 추가적 클린코드는 다음 Step에서 진행할 예정입니다.

kses1010 committed:
Step1 Hello world Git push test

kses1010 committed:
Intellij 빌드 되지 않기때문에 xml 파일 추가

honux77 reviewed (changes requested):
좋은 커밋 메시지 글 다시 읽고 커밋 메시지 변경해서 제출해 주세요.

kses1010 committed:
리드미 Step1 수정 및 MonsterRaceMain 다시 시작

리드미에서 띄어쓰기 수정하고 나쁜 커밋메시지로 인해 처음부터 다시
시작합니다. git reset --hard를 입력하고 코드를 작성하기 전으로 되돌렸습니다.
MonsterRaceMain에서 main만 만든 상태입니다.

kses1010 committed:
몬스터 레이스 스켈레톤 코드 작성

몬스터레이스 간단한 출력부와 TODO를 작성하여 앞으로 해야할 작업에
대해서 간단히 코드 작성했습니다.

kses1010 committed:
몬스터 숫자와 시도할 횟수 입출력 및 레이스출력메소드 추가

kses1010 committed:
makeRandomCount, conditionalForward, racePrint 작성

makeRandomCount는 랜덤숫자 생성, conditionalForward 조건부 전진
racePrint는 race를 출력합니다. racePrint는 이중 for문으로 이루어져
있으며 추가적인 코드추가가 필요합니다.

kses1010 committed:
racePrint 메소드 작성완료 Stpe1 게임 작동완료

racePrint메소드에서 print로 conditionalForward 메소드 적용
실행을 그대로 해보니 제대로 출력이 됩니다.

kses1010 commented:
처음부터 다시 시작했습니다. 커밋메시지를 여기서 더 구체적으로 진행하면 되나요?

honux77 commented on src/codesquad/sunny/race/MonsterRaceMain.java:
```diff
@@ -0,0 +1,48 @@
+import java.util.Random;
+
+public class MonsterRaceMain {
+    private static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
```
가급적이면 static은 사용하지 말아 주세요.

honux77 commented on src/codesquad/sunny/race/MonsterRaceMain.java:
```diff
@@ -0,0 +1,48 @@
+    private static int monsterCount;
+    private static int tryCount;
+
+    public static void main(String[] args) throws IOException {
```
main함수에 구현 로직은 빼 주세요. 
별도의 클래스와 클래스의 메소드로 분리해 주세요.

honux77 reviewed (approved):
수고하셨어요! 👏
